libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -393,9 +532,11 @@ void AC_PrecLand::check_ekf_init_timeout()
         if (AP_HAL::millis()-_last_update_ms > EKF_INIT_SENSOR_MIN_UPDATE_MS) {
             // we have lost the target, not enough readings to initialize the EKF
             _estimator_initialized = false;
+            gcs().send_text(MAV_SEVERITY_CRITICAL, "PrecLand: Initialization Failed");
```

Comment:
let's save some bytes and rename this to "PrecLand: Init Failed"

Reply:
we could also save some bytes by defining a const char* precland_prefix = "PrecLand: " and then 

>> gcs().send_text(MAV_SEVERITY_CRITICAL, "%sInit Failed", precland_prefix);

not a blocker though